src/angular/2-building-first-app/building-first-app.md

```diff
@@ -258,8 +219,8 @@ We also included `schemas` metadata for our module. <a href="https://angular.io/
 
 ✏️ Update **src/app/app.component.spec.ts**:
 
-@sourceref ./app.component.spec.ts
-@highlight 37
+@sourceref ../../../exercises/angular/2-building-first-app/problem/src/app/app.component.spec.ts
+@highlight 1,15-17,37,only
```

Comment:
I think this makes the highlighting nicer:

<img width="718" alt="Diff with NO_ERRORS_SCHEMA highlighted." src="https://github.com/bitovi/academy/assets/10070176/1b188834-09fd-4e58-9037-cb603b13df32">

Reply:
looks good